Docs: Adding panel suggestions initial documentation for 8.3 (#42488)
grafana/grafana: docs/sources/visualizations/_index.md

Grafana offers a variety of visualizations to support different use cases. This section of the documentation highlights the built-in panels, their options and typical usage.

> > **Note:** If you are unsure which visualization to pick, Grafana can provide visualization suggestions based on the panel query. Additionally, Grafana will display a preview of the selected visualization type or visualization suggestion. Please see the [add a panel]({{< relref "../pnaels/add-a-panel.md" >}}) documentation for more info.
> 

Reviewer: ```suggestion
> **Note:** If you are unsure which visualization to pick, Grafana can provide visualization suggestions based on the panel query. When you select a visualization, Grafana will show a preview with that visualization applied.  For more information, see the [add a panel]({{< relref "../pnaels/add-a-panel.md" >}}) documentation.

```
